timeline.tex

```diff
@@ -1,22 +1,49 @@
 \section{Roadmap and Timeline} \label{sec:timeline}
 
-This timeline provides a list of key dates  related to the \es program. 
-It will be updated with more dates related to the development of the \esp as they are defined. 
-
-\textbf{2021-11-12:} Issue first version of the \ro Plans for an \esp. 
-
-\textbf{2023-01-20:} Commissioning Camera (ComCam) on sky, engineering first light.
-
-\textbf{2023-07-01:} LSST Science Camera (LSSTCam) on sky.
-
-\textbf{2023-10-01:} Start of the \svs.
-
-\textbf{2023-12-18:} Current earliest completion date for construction.
-
-\textbf{2024-07-01:} Current latest completion date for construction.
-
-\textbf{2024-04-01:} Planned start of Operations Phase.
-
-\textbf{2024-04/09:} Current forecast date for Data Preview 2, LSSTCam On-Sky Data.
-
-\textbf{2025-02/07:} Current forecast date for Data Release 1.
\ No newline at end of file
+The timeline shown in Table~\ref{tab:milestones} below provides a list of key dates related to the \es program.
+It will be updated with more dates related to the development of the \esp as they are defined.
+
+The date ranges in the Table are derived from the ``Celebratory Milestones'' list published at monthly intervals on the Rubin Project website;\footnote{\url{https://www.lsst.org/about/project-status}} shown here are the milestones as of October 2022, which use Project controls data from August 2022.
+
+Milestone dates are given as min-max ranges to indicate their uncertainty.
+Typically the near date corresponds to the current Project forecast, plus any additional operational uncertainty.
+The late date corresponds (approximately) to the current Project ``late date'' plus any additional operational uncertainty: this late date cannot be surpassed without the Project rebaselining its schedule.
+
+The late dates for the DP2 and DR1 data release milestones allow for the possibility that the Project completes within its late date, but in doing so reduces the amount of on-sky LSST Cam commissioning time.
+{\it In this eventuality, the operations team would spend up to 3 months at the start of the full/survey operations phase completing any remaining Science Validation Survey observations, such that DP2 could be realized as planned.}
+The LSST survey will start shortly after the completion of the SV surveys.
+
+As can be seen in the table, the LSST survey is currently expected to start sometime between Summer 2024 and Spring 2025.
```

Comment:
Suggest to use language that is more inclusive to our colleagues in the southern hemisphere.

Reply:
(maybe "mid-2024" and "early 2025"?)

Reply:
I was suddenly embarrassed that I, an antipodean, did this, then I remembered Phil wrote this part :)